src/main/java/com/rqlite/Rqlite.java

```diff
@@ -3,6 +3,7 @@
 import com.rqlite.dto.ExecuteResults;
 import com.rqlite.dto.Pong;
 import com.rqlite.dto.QueryResults;
+import com.rqlite.impl.NodeUnavailableException;
```

Comment:
`impl` in the package name implies that details of those package should not be exposed. Shouldn't this exception be defined elsewhere?